networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -230,19 +251,60 @@ def cleanup_invalid_cfg(self, hd, routers):
         cfg_syncer = syncer.ConfigSyncer(routers, self, hd)
         cfg_syncer.delete_invalid_cfg()
 
+    def _create_vrf(self, ri):
```

Comment:
don't need this _create_vrf() and _remove_vrf() call as it should be just same as parent's per our discussion.